Docker build failing since alpine:3.14

Dockerfile uses `FROM alpine` which always pulls the `:latest` tag. In June 2021 alpine latest tag changed from 3.13 to 3.14 and now the builds are failing:

```
...
Step 4/14 : RUN 	cd /tmp/beanstalkd && 	make
--
135 | ---> Running in 21e33e2c80bf
136 | make: /bin/sh: Operation not permitted
137 | make: /bin/sh: Operation not permitted
138 | make: /bin/sh: Operation not permitted
139 | make: /bin/sh: Operation not permitted
140 | make: /bin/sh: Operation not permitted
141 | make: /bin/sh: Operation not permitted
142 | make: ./verc.sh: Operation not permitted
143 | make: /bin/sh: Operation not permitted
144 | cc   dat.h   -o .o
145 | make: cc: Operation not permitted
146 | make: *** [<builtin>: .o] Error 127
147 | The command '/bin/sh -c cd /tmp/beanstalkd && 	make' returned a non-zero code: 2
148 | time="2021-06-27T08:02:19Z" level=fatal msg="exit status 2"
```

> Dockerfile uses `FROM alpine` which always pulls the `:latest` tag. In June 2021 alpine latest tag changed from 3.13 to 3.14 and now the builds are failing:
> 
> ```
> ...
> Step 4/14 : RUN 	cd /tmp/beanstalkd && 	make
> --
> 135 | ---> Running in 21e33e2c80bf
> 136 | make: /bin/sh: Operation not permitted
> 137 | make: /bin/sh: Operation not permitted
> 138 | make: /bin/sh: Operation not permitted
> 139 | make: /bin/sh: Operation not permitted
> 140 | make: /bin/sh: Operation not permitted
> 141 | make: /bin/sh: Operation not permitted
> 142 | make: ./verc.sh: Operation not permitted
> 143 | make: /bin/sh: Operation not permitted
> 144 | cc   dat.h   -o .o
> 145 | make: cc: Operation not permitted
> 146 | make: *** [<builtin>: .o] Error 127
> 147 | The command '/bin/sh -c cd /tmp/beanstalkd && 	make' returned a non-zero code: 2
> 148 | time="2021-06-27T08:02:19Z" level=fatal msg="exit status 2"
> ```

you may find it on https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,21 +1,13 @@
+ARG BASE=alpine
 FROM alpine as builder
-
-RUN \
-	apk -U upgrade --no-cache && \
-	apk add --no-cache build-base git 
-
+RUN apk add --no-cache build-base git
 COPY . /tmp/beanstalkd
-RUN \ 
-	cd /tmp/beanstalkd && \
-	make 
+RUN cd /tmp/beanstalkd && make
 
 ################################
-FROM alpine
-
-RUN apk -U upgrade --no-cache 
+ARG BASE
+FROM ${BASE}
 
 COPY --from=builder /tmp/beanstalkd/beanstalkd /usr/bin/
-
-RUN mkdir /beanstalkd
 EXPOSE 11300
 ENTRYPOINT ["/usr/bin/beanstalkd"]

